chemclipse/plugins/org.eclipse.chemclipse.chromatogram.xxd.peak.detector.supplier.firstderivative/src/org/eclipse/chemclipse/chromatogram/xxd/peak/detector/supplier/firstderivative/core/PeakDetectorMSD.java

```diff
@@ -159,12 +164,12 @@ private void buildAndStorePeaks(List<IRawPeak> rawPeaks, IChromatogramMSD chroma
 	 * @param windowSize
 	 * @return {@link IFirstDerivativeDetectorSlopes}
 	 */
-	public static IFirstDerivativeDetectorSlopes getFirstDerivativeSlopes(IChromatogramSelectionMSD chromatogramSelection, WindowSize windowSize) {
+	public static IFirstDerivativeDetectorSlopes getFirstDerivativeSlopes(IChromatogramSelectionMSD chromatogramSelection, IPeakDetectorSettingsMSD peakDetectorSettings) {
```

Comment:
See comment above, the method does not need the settings, it should contain only 

- WindowSize windowSize 
- IMarkedIons filter

as parameters

Reply:
Resolved with 96ddd0625efcb7073a2c87dd4f8eb46bee8e2ec6.